### PR TITLE
Make auditlog optional

### PR DESCRIPTION
**Pull request by rakesh-garimella**
**Description**

Changes proposed in this pull request:

- Have a flag to enable/disable Auditlog
- Pass the secrets in fluent bit pod using environment variable.

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/auditlog/config.go [hunk 1]
```diff
@@ -6,4 +6,5 @@ type Config struct {
 	Password      string `envconfig:"APP_AUDITLOG_PASSWORD"`
 	Tenant        string `envconfig:"APP_AUDITLOG_TENANT"`
 	EnableSeqHttp bool   `envconfig:"APP_AUDITLOG_ENABLE_SEQ_HTTP"`
+	Disabled      bool   `envconfig:"default=false"`
```
Comment (skhalash) [inline components/kyma-environment-broker/internal/auditlog/config.go:9]: This annotation looks different from the rest in the struct. Will it work?

Comment (rakesh-garimella) [reply components/kyma-environment-broker/internal/auditlog/config.go:9]: The `envconfig` accepts default value: https://github.com/vrischmann/envconfig#default-values

**Review by skhalash — DISMISSED** [review]

**Comment by rakesh-garimella** [comment]
Blocked because of following issue: https://github.com/kyma-project/control-plane/issues/708

**Review by suleymanakbas91 — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Comment by rakesh-garimella** [comment]
/test pre-main-control-plane-gke-integration



<br class="Apple-interchange-newline">

**Pull request merged**